Formula/stlink.rb

```diff
@@ -18,7 +18,12 @@ class Stlink < Formula
   depends_on "libusb"
 
   def install
-    system "cmake", ".", *std_cmake_args
+    args = std_cmake_args
+    on_linux do
+      args << "-DSTLINK_MODPROBED_DIR=#{lib}/modprobe.d"
+      args << "-DSTLINK_UDEV_RULES_DIR=#{lib}/udev/rules.d"
```

Comment:
Alternatively can use `#{etc}` vs `#{lib}`. Not sure if there is a preference.

Reply:
I think the standard is that configs installed by the package manager like we are doing go in `lib` while user modifications would go in `etc`, so `lib` makes sense here.

Reply:
I think we install all configs in `etc`. Installing them into `etc` means that users can modify them without `brew` destroying their changes on an upgrade/reinstall.

Reply:
So is `etc` the preference here?

Formulae `udev/rules.d` usage is currently split:
- `libfuse` & `libfuse@2` use `#{etc}/udev/rules.d` 
- `hackrf` & `libfido2` use `#{lib}/udev/rules.d`

Reply:
It depends. If it's a config file that users are supposed to be able to modify, then install them into `etc`. Otherwise we can keep them in `lib`.

Reply:
I wonder if udev/modprobe configs are even usable in the Linuxbrew default directory without copying the files into a system-specific directory.

If the user will end up copying the files anyway, then it may be better to just leave it in `lib` to automatically remove on uninstall.